DatePicker/tests/DatePicker-specs.js

```diff
@@ -161,4 +191,10 @@ describe('DatePicker', () => {
 
 		expect(header).toBeNull();
 	});
+
+	test('\'dateToLocaleString\' method should return \'null\' for an \'undefined\' date', () => {
+		const actual = dateToLocaleString(undefined);		// eslint-disable-line no-undefined
+
+		expect(actual).toBeNull();
+	});
```

Comment:
```suggestion
	describe('#dateToLocaleString', () => {
		test('should return \'null\' for an \'undefined\' date', () => {
			const actual = dateToLocaleString(undefined);		// eslint-disable-line no-undefined

			expect(actual).toBeNull();
		})
	})

```
I think we can separate `dateToLocaleString` method with describe.

Reply:
Done, and I applied the same changes for timeToLocaleString inside TimePicker.